tools/pd-simulator/simulator/cases/balance_region.go

```diff
@@ -36,7 +36,7 @@ func newRedundantBalanceRegion() *Case {
 
 	for i := 0; i < storeNum; i++ {
```

Comment:
if we add/down-node the case checker should check all the stores balance , but it still use storeNum.
it may exists somewhere else.